simpleflow/utils/__init__.py

```diff
@@ -4,7 +4,7 @@
 from zlib import adler32
 
 from . import retry  # NOQA
-from .json_tools import json_dumps, json_loads_or_raw, serialize_complex_object  # NOQA
+from ._json import json_dumps, json_loads_or_raw, serialize_complex_object  # NOQA
```

Comment:
Not sure this one should be private?

Reply:
No strong opinion, I can change!

Reply:
We use some of these functions in our codebase (OK, with a `from simpleflow.utils import`; but what if we decide to do the same one day 😛)

Reply:
(I'm not sure that it makes any sense whatsoever that anyone would be piggybacking on anexternal lib's internal utils. I vote for "we actively discourage it" here.)

Reply:
We should have extracted this beautiful "json.dumps" improvement in its own package, NPM style. (joking)

Reply:
I changed it but left as a distinct commit. After that I find the inconsistency in utils imports a bit ugly, so I'd rather remove the commit and leave as it was in the first iteration. It's a matter of taste I guess, I won't fight 😉

Reply:
(Sorry for the late answer) Do as you prefer 🙂

Reply:
I removed the latter commit and consider this thread resolved 😉 Bye `json_tools.py`, long live `_json.py`